### ISSUE TITLE
Bounty Paid event request

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#788]
## EDDI version in which issue found
v3.0.1 under VA 1.7.2

## Steps to reproduce
Pay a Bounty Incurred

## Expected
An event triggers for this, similar to paying Fines.

## Observed
There is no event to trigger in EDDI.

## Investigation
This is something I've hoped would be added since the Crime & Punishment update, so I can add it to my Fines and Bounties functionality.  Any chance this can be created please?

Here's a section of the Journal regarding this event:-

`{ "timestamp":"2018-08-12T02:05:27Z", "event":"PayBounties", "Amount":300, "Faction":"$faction_Independent;", "Faction_Localised":"Independent", "ShipID":22, "BrokerPercentage":25.000000 }`

I've already created a temporary 'placeholder' script in my personality:-

```
{_ Context }
{SetState('eddi_context_last_subject', 'bounty')}
{SetState('eddi_context_last_action', 'pay')}
{SetState('eddi_context_fine_amount', event.amount)}

{_ This is not a default EDDI event as of v3.0.1-b1. _}
{_ This script will do nothing until EDDI is updated to include this event. _}
{_ It is included here as an example, and should work if EDDI has this event in the future. _}

{_ Update Fines & Bounties}
{F("FinesBounties_Delete")}

Paid bounty of {Humanise(event.amount)} credits.

{if event.brokerpercentage:
  {Pause(1000)}
  The Broker has taken {event.brokerpercentage} credits.
}
```
Obviously, you won't need the note, or the call to my FinesBounties_Delete function script.

**Comment by Tkael** [comment]
This is already in the to-do list under issue #378, though perhaps it needs a little more prominence.

**Comment by Darkcyde13** [comment]
My apologies.  I did try a search for 'paying bounties' (and combinations thereof), but they came up empty.

**Comment by Tkael** [comment]
Separated out from #378 for more individual focus.

**Comment by Darkcyde13** [comment]
Bah, in my usual haste and thinking three steps ahead of myself, I now realise my previous comment doesn't quite add up and look right.

What I meant was that the calculation:-
`{round((event.amount / (100 - event.brokerpercentage)) * event.brokerpercentage)}`
doesn't work out correctly.

Using my example of 10650 reward:-
{round((7987 / (100 - 25)) * 25)} = 2662
7987 (the actual reward) + 2662 = 10649

However, using 'ceil' instead (I got them the wrong way around above):-
{ceil(7987 / (100 - 25)) * 25)} = 2663
7987 + 2663 = 10650

This is for collecting at I.F.  For paying, I think it needs to be 'floor' as the game will round up the amount you pay, so the broker amount needs to be rounded down.

Admittedly, I've only tried with collecting bounties.  I need to try getting and paying fines sometime today if I can find the time.
